src/@types/pkg.ts

```diff
@@ -75,6 +75,7 @@ export interface RawPkg {
   moduleTypes: ModuleType[];
   lastCrawl: string;
   _searchInternal: {
+    expiresAt: string;
```

Comment:
if we want to use it for a filter, why isn't it a timestamp/number?

Reply:
same same, it's just easier to read as string no?